R/crunchbox.R

```diff
@@ -5,11 +5,16 @@
 #' @param dataset CrunchDataset
 #' @param filters FilterCatalog, or \code{NULL} for no filters. Default all
 #' filters in your catalog, \code{filters(dataset)}.
+#' @param brand_colors an optional list with colors (in hex format) named 'primary', 'secondary', and 'message'
+#' @param static_colors an optional list of static colors (in hex format) to use for categoricals (taken in order)
+#' @param category_color_lookup an optional list of category names to colors (in hex format) to use for that category (for fine grained tuning of category colors.)
 #' @param ... additional metadata for the box, such as "title", "header", etc.
 #' @return The URL to the newly created box.
 #' @seealso \code{\link{preCrunchBoxCheck}} to provide guidance on what you're including in the CrunchBox
 #' @export
-crunchBox <- function (dataset, filters=crunch::filters(dataset), ...) {
+crunchBox <- function (dataset, filters=crunch::filters(dataset),
+                       brand_colors, static_colors,
```

Comment:
I think it would be helpful to have an example of changing the colour scheme. You can probably just copy the ones from the tests.

Reply:
👍